.husky/prepare-commit-msg

```diff
@@ -0,0 +1,4 @@
+[ -n "$CI" ] && exit 0
+
+exec < /dev/tty
+node ./lib/cli.js --hook || true
```

Comment:
This is for the local development of `git-cz`.

May be helpful to add in README that if you install `git-cz` locally and not via `-g` and are using `husky@5+` in your project that imports `git-cz` you should heed:

**Locally installed binaries**
- If you were calling directly locally installed binaries, you need to run them via your package manager:
- ref: https://typicode.github.io/husky/#/?id=locally-installed-binaries

Which would upgrade to:

```sh
npx --no-install git-cz --hook || true
```
```sh
yarn git-cz --hook || true
```

Reply:
🤕  I skipped that in the documentation on my imports and caused a lot of needless head-scratching and back-and-forth testing.